GPU/Common/GPUDefGPUParameters.h

```diff
@@ -99,7 +100,8 @@
   #define GPUCA_LB_GPUTPCGMMergerResolve_step0 256
   #define GPUCA_LB_GPUTPCGMMergerResolve_step1 256
   #define GPUCA_LB_GPUTPCGMMergerResolve_step2 256
-  #define GPUCA_LB_GPUTPCGMMergerResolve_step3 256, 4
+  #define GPUCA_LB_GPUTPCGMMergerResolve_step3 256
+  #define GPUCA_LB_GPUTPCGMMergerResolve_step4 256, 4
```

Comment:
The #ifndef part below is missing for step 4

Reply:
Fixed. And did some minor code cleanup.